src/jquery.sceditor.js

```diff
@@ -69,8 +69,8 @@ define(function (require) {
 
 		options = options || {};
 
-		if (!options.runWithoutWysiwygSupport && !browser.isWysiwygSupported) {
-			return;
+		if (options.runWithoutWysiwygSupport || !browser.isWysiwygSupported) {
+			SCEditor.isWysiwygSupported = false;
```

Comment:
If it's `true` it should force the editor to run in source mode without WYSIWYG support. If it's `false` the editor should be disabled.

With `(!options.runWithoutWysiwygSupport && !browser.isWysiwygSupported)` both WYSIYWG support and `runWithoutWysiwygSupport` have to be false for it to not run. If the editor runs and `isWysiwygSupported` is `false` then it locks into source mode.